modules/nw-networkpolicy-audit-configure.adoc

```diff
@@ -0,0 +1,229 @@
+[id="nw-networkpolicy-audit-configure_{context}"]
+= Configuring network policy auditing
+
+As a cluster administrator, you can customize network policy audit logging to suit your environment.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
+* To customize the network policy audit logging configuration, enter the following command:
++
+[source,terminal]
+----
+$ oc edit network.operator.openshift.io/cluster
+----
++
+[TIP]
+====
+You can alternatively apply the following YAML to configure audit logging:
```

Comment:
@jboxman The operator configures the OVN cluster with the values listed by default. The user does not need to do basic configuration but values can be changed for destination, maxFileSize, rateLimit and syslogFacility. The user just enables network policy auditing on the namespace.